go.mod

```diff
@@ -1,19 +1,18 @@
 module riverqueue.com/riverui
 
-go 1.22.0
+go 1.23.0
```

Comment:
Ugh, I tried not to change this, but the new repo _really_ wanted to be Go 1.23 (and run of `go mod tidy` would change it to such), and when I brought that in here, of course it'd take this project to 1.23 as well.

Probably okay though since we don't really need to make as much an effort to support very old Go versions for this project, and currently only 1.23 and 1.24 are supported.

Reply:
Yeah I think it's fine, latest 2 Go versions is a reasonable and common stance.

Reply:
Yep, +1. SG.